src/app/features/groups/components/group-card/group-card.component.ts

```diff
@@ -0,0 +1,42 @@
+import { ChangeDetectionStrategy, Component, EventEmitter, Input, Output } from '@angular/core';
+import { Group } from '../../models/group'; 
+import { Translations } from 'src/app/core/services/translations.service';
+import { MaterialButtonType } from 'src/app/shared/models/material-button-type';
+import { Router } from '@angular/router';
+import { CoreRoutes } from 'src/app/core/constants/routes.constants';
+import { TooltipDirection } from 'src/app/shared/models/tooltip-direction';
+
+ @Component({
+   selector: 'app-group-card',
+   templateUrl: './group-card.component.html',
+   styleUrls: ['./group-card.component.scss'],
+   changeDetection: ChangeDetectionStrategy.OnPush
+ })
+ export class GroupCardComponent {
+   @Input() model?: Group;
+   @Input() loading: boolean = false;
+
+   materialButtonType = MaterialButtonType;
+   tooltipDirection = TooltipDirection;
```

Comment:
I don't think we'll be needing tooltips for this component. You can remove this.